packages/patternfly-4/react-core/src/components/Pagination/Navigation.tsx

```diff
@@ -125,6 +127,15 @@ export class Navigation extends React.Component<NavigationProps, NavigationState
       this.state.userInputPage !== this.props.page
     ) {
       this.setState({ userInputPage: this.props.page });
+      if (this.inputRef) {
+        if (this.props.lastPage >= 100) {
+          const charLength = String(this.props.lastPage).length;
+          this.inputRef.current.style.setProperty(
+            '--pf-c-pagination__nav-page-select--c-form-control--width-chars',
+            String(charLength)
```

Comment:
What happens without casting to a string? I think `setProperty` will still work.